tests/README.md

```
@@ -3,101 +3,151 @@
+    docker/          Docker Compose files for local backend services
     framework/       the framework unit tests
     runtime/         the application runtime dir for the yii test app
 
 ## HOW TO RUN THE TESTS
 
-Make sure you have PHPUnit installed and that you installed all composer dependencies (run `composer update` in the repo base directory).
+Make sure all Composer dependencies are installed in the repository root.
 
 Run PHPUnit in the yii repo base directory.
 
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)